examples/with-solidstart/.env.example

```
@@ -0,0 +1,2 @@
+# Generate using `openssl rand -hex 32`
+SESSION_SECRET = myverylongsessionsecretkeythatishouldchange
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)